File: parsers/openapi.go
package parsers

import (
	s "apivapt/schema"
	"net/http"
)

func (o *OpenAPIParser) Detect(header http.Header, body []byte) bool {
	return false
}

func (o *OpenAPIParser) HasRoutes(data []byte) bool {
	return false
}

func (o *OpenAPIParser) Parse(data []byte) (*s.APISchema, error) {
	return nil, nil
}

File: parser/openapi.py
import httpx

from schema.schema import APISchema
from .base import BaseParser


class OpenAPIParser(BaseParser):
    def Detect(self, base_url: str, header: httpx.Headers, body: str) -> tuple[bool, str]:
        return False, ""

    def HasRoutes(self, data: str) -> bool:
        return False

    def Parse(self, data: str) -> APISchema:
        return APISchema()
